tests/stack.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2021 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import itertools
+
+import numpy as np
+
+import cunumeric as num
+
+
+def run_test(arr, test_routine):
+    input_arr = []
+    for axis in range(arr[0].ndim):
+        input_arr.append(axis)
+
+    for routine, input_opt in itertools.product(test_routine, input_arr):
+        print_msg = f"np.{routine}((array({arr[0].shape}) * 3), {input_opt}"
+        b = getattr(np, routine)(arr, input_opt)
```

Comment:
I feel this is an unnecessary complication, knowing that you only test `np.concatenate` in this code; this just makes it difficult to track what's being tested here.